data/strings/strings.txt

```diff
@@ -1131,8 +1131,8 @@
     tr = • Hızlı, minimalist ve hafif
     uk = • Швидкий, мінімалістичний, без зайвого сміття
     vi = • Nhanh chóng, tối giản, không có rác
-    zh-Hans = •快速，简约，没有垃圾
-    zh-Hant = •快速，簡約，沒有垃圾
+    zh-Hans = •快速，简约，不含垃圾软件
```

Comment:
"Software bloat" may be translated as "软件膨胀", and "垃圾软件" which means "junk software". Therefore, "no software bloat" should be translated as "软件不会自我膨胀". However, since the whole sentence implies a subject of "This software/app", only the phrase "不会自我膨胀" is sufficient.

---

“software bloat”或许翻译成 “软件膨胀” 更好，“垃圾软件”的意思是 “junk software”
因此"no software bloat"应为“软件不会自我膨胀”，但因为整句话中暗含一个“This software/app”作为主语，因此直接仅留下“不会自我膨胀”六个字即可